### DISCUSSION TITLE
Table 'progro' already exists

### DISCUSSION THREAD

**Discussion by erkin98** [sqlalchemy/sqlalchemy#6330] [Usage Questions]
When i use to upload my dataframe to database i get this error
``` raise errors.get_mysql_exception(exc.errno, msg=exc.msg,
sqlalchemy.exc.ProgrammingError: (mysql.connector.errors.ProgrammingError) 1050 (42S01): Table 'progro' already exists
[SQL:
CREATE TABLE progro (
        job TEXT,
        job_descr TEXT,
        company TEXT,
        city TEXT,
        job_type TEXT,
        wage FLOAT(53),
        deadline FLOAT(53),
        links TEXT
)

]
(Background on this error at: http://sqlalche.me/e/14/f405)

how can i fix this?

**Comment by erkin98** [comment]
when i use 1.3 version i dont see this error but when updating sqlalchemy and use 1.4 version this i encountered with this error

**Comment by CaselIT** [comment]
Hi,

Without an example of how you are getting this error is really hard to suggest any solution

**Reply by zzzeek** [reply]
I think this almost certainly is what we addressed in b26cf96462b195a4c12ccdf8283.

looking at pandas, here's where they seem to be doing somethign related to SQLAlchemy 1.4 now: https://github.com/pandas-dev/pandas/blob/49c779118c2d473c60565d9010e119bb62d06f74/pandas/io/sql.py#L1630 and the previous code is the buggy version so this is likely https://github.com/pandas-dev/pandas/pull/40471   .   I would upgrade pandas to the latest version

**Reply by erkin98** [reply]
i downgraded to 1.3 and surprisely its worked

**Comment by zzzeek** [comment]
This seems to be a pandas issue:

Here's where they seem to be doing somethign related to SQLAlchemy 1.4 now: https://github.com/pandas-dev/pandas/blob/49c779118c2d473c60565d9010e119bb62d06f74/pandas/io/sql.py#L1630 and the previous code is the buggy version so this is likely pandas-dev/pandas#40471 . I would upgrade pandas to the latest version

**Reply by erkin98** [reply]
i get same error today 
sqlalchemy 1.4.11
pandas 1.2.4
